client/src/plus/app.tsx

```diff
@@ -1,56 +1,65 @@
-import React, { useEffect } from "react";
-import { useLocale } from "./../hooks";
+import React from "react";
+import Modal from "react-modal";
+import { useLocale } from "../hooks";
 
 import "./fonts/metropolis.css";
 import "./fonts/inter.css";
 import "./index.scss";
 
+Modal.setAppElement("main");
+
+function BackToMDNModal({ onRequestClose }) {
+  const locale = useLocale();
+
+  return (
+    <Modal
+      overlayClassName="modal"
+      className="modal-inner"
+      aria-expanded
+      onRequestClose={onRequestClose}
+      isOpen
+      parentSelector={() => document.querySelector(".plus main")!}
+    >
+      <div className="modal-inner">
+        <div className="modal-content">
+          <figure className="dino" />
+          <h1>Thank you for your interest in MDN Plus</h1>
+          <p>
+            We are grateful for all the feedback you sent our way. The waitlist
+            signup and survey are now closed, and we’re beginning work towards
+            an initial release.
+          </p>
+          <p>We'll be in touch soon.</p>
+          <a href={`/${locale}/`}>Back to MDN</a>
+        </div>
+        <button
+          type="button"
+          onClick={() => {
+            onRequestClose();
+          }}
+          aria-label="Close modal"
+        >
+          ✖
+        </button>
+        <figure className="mandala" aria-hidden="true" />
```

Comment:
Same comment here. Also, when it is purely decorative, adding an empty `alt` attribute to an `img` element will have the same effect for screen readers as setting `aria-hidden`. It is also possible to use multiple background images on the same HTML element so, perhaps that is also an option to explore?

The other benefit of using the `img` element is that we have access to the `decoding=async` and `loading=lazy` attributes. Not sure if these would be useful in this context but, thought I would just throw it out there as a consideration.